text/0039-sdk-performance-issue-creation.md

```diff
@@ -0,0 +1,67 @@
+* Start Date: 2022-11-28
+* RFC Type: decision
+* RFC PR: [#33](https://github.com/getsentry/rfcs/pull/33)
+* RFC Status: draft
+* RFC Driver: [Philipp Hofmann](https://github.com/philipphofmann)
+
+# Summary
+
+This RFC aims to clarify which types of performance issues SDKs shall create.
+
+# Motivation
+
+On June 21, 2022, we decided with [DACI](https://www.notion.so/sentry/Performance-Issue-Creation-POC-e521772ebccb482b83b08f4f8a3db2cb) to create performance issues in Ingest. While implementing the file I/O on the main thread performance issue for Android, the question arose as to why SDKs don't report easy-to-detect and matured performance issues. 
+
+# Options Considered
```

Comment:
Options 1 and 2 are missing some Cons which is why the Ingest side was chosen

1. The detectors must be maintained across all SDKs that detect a specific issue.

e.g. N+1 in Django, Rails, Node, Laravel, etc...
File I/O in both Android and iOS.

2. Stack traces are inherently expensive to process.